sdks/python/gen_protos.py

```diff
@@ -216,6 +216,23 @@ def _import(m):
     sys.path.pop(0)
 
 
+def _find_protoc_gen_mypy():
```

Comment:
Do you want to keep this code in?

Reply:
Yeah, might as well keep it.  It includes a fallback when PATH is not set and it’s easier to debug.

Reply:
Yeah, might as well keep it.  It includes a fallback when PATH is not set and it’s easier to debug.